src/agora/dodemul.hpp

```diff
@@ -17,7 +17,9 @@
 #include <string.h>
 #include <vector>
 
-#define USE_MKL_JIT 0 // JIT for cgemm is only available after MKL 2019 update 3
+// Just-in-time optimization for MKL cgemm is available only after MKL 2019
+// update 3. Disable this on systems with an older MKL version.
+#define USE_MKL_JIT 1
```

Comment:
I think it is better to make JIT disabled by default because newer version of Intel compiler has some conflicts with FlexRAN SDK 19.04.

Reply:
I pushed a change to configure USE_MKL_JIT based on the MKL version.